go.sum

```diff
@@ -1,16 +1,33 @@
+github.com/google/go-cmp v0.2.0/go.mod h1:oXzfMopK8JAjlY9xF4vHSVASa0yLyX7SntLO5aqRK0M=
 github.com/google/go-cmp v0.3.0 h1:crn/baboCvb5fXaQ0IJ1SGTsTVrWpDsCWC8EGETZijY=
 github.com/google/go-cmp v0.3.0/go.mod h1:8QqcDgzrUqlUb/G2PQTWiueGozuR1884gddMywk6iLU=
+github.com/google/go-cmp v0.3.1/go.mod h1:8QqcDgzrUqlUb/G2PQTWiueGozuR1884gddMywk6iLU=
+github.com/jsimonetti/rtnetlink v0.0.0-20190606172950-9527aa82566a/go.mod h1:Oz+70psSo5OFh8DBl0Zv2ACw7Esh6pPUphlvZG9x7uw=
```

Comment:
Is this used? Can you run `go mod tidy` please? Alternatively, try reverting your changes to go.sum and rebuild.

Reply:
run `go mod tidy`
```
go mod tidy
go: downloading github.com/jsimonetti/rtnetlink v0.0.0-20190606172950-9527aa82566a
go: extracting github.com/jsimonetti/rtnetlink v0.0.0-20190606172950-9527aa82566a
```